userpatches/customize-image.sh

```diff
@@ -260,11 +284,72 @@ apt-get install -y evcc
 
 # Pre-generate minimal config if missing
 if [[ ! -f /etc/evcc.yaml ]]; then
-  cat >/etc/evcc.yaml <<YAML
+	if [[ "$OPENWB" == "true" ]]; then
+		cat >/etc/evcc.yaml <<YAML
 network:
   schema: https
   host: ${EVCC_HOSTNAME}.local
+
+# Device used for meters and chargers can be either /dev/ttyUSB0 or /dev/ttyACM0 depending on installed modbus converter
+meters:      # Uncomment one of the following meters depending on which one is installed in your OpenWB
+- type: template
+  template: abb-ab
+  id: 201
+  device: /dev/ttyUSB0
+  baudrate: 9600
+  comset: 8N1
+  usage: charge
+  modbus: rs485serial
+  name: openwb-meter
+#- type: template
+#  template: mpm3pm
+#  id: 5
+#  device: /dev/ttyUSB0
+#  baudrate: 9600
+#  comset: 8N1
+#  usage: charge
+#  modbus: rs485serial
+#  name: openwb-meter
+#- type: template
+#  template: eastron
+#  id: 105
+#  device: /dev/ttyUSB0
+#  baudrate: 9600
+#  comset: 8N1
+#  usage: charge
+#  modbus: rs485serial
+#  name: openwb-meter
+
+chargers:
+- type: template
+  template: openwb-native
+  modbus: rs485serial
+  id: 1                   # EVSE is on Modbus Id 1
+  device: /dev/ttyUSB0
+  baudrate: 9600
+  comset: 8N1
+  name: openwb-charger
+  phases1p3p: true
+
+loadpoints:
+- title: MyLoadpoint
+  charger: openwb-charger
+  meter: openwb-meter
+
+site:
+  title: MyHome
+YAML
+
+		# Add necessary groups to allow user evcc to access OpenWB HW
+		usermod -aG dialout,input,gpio evcc
+
+	else
+		cat >/etc/evcc.yaml <<YAML
+network:
+  schema: httpslocal
+  host: ${EVCC_HOSTNAME}.
```

Comment:
Why is `httpslocal` differnt here? Can´t we get rid of the `else` case by always creating the network section and only conditionally append the openwb block.